lib/src/segy.c

```diff
@@ -711,8 +739,13 @@ int segy_samples( const char* binheader ) {
 
 int segy_trace_bsize( int samples ) {
```

Comment:
I don't find it intuitive to have both "segy_trace_bsize" and "segy_trsize", and the difference of the functions by just looking at the names?

Reply:
"By default, for backwards compatilibity and convenience, 4-byte samples
are assumed." <-- backwards compatibility is the reason?

Reply:
Yes, backwards compatibility is the sole reason, otherwise I'd reuse the trace_bsize name. This way, old code keeps working as it has, and only new code targeting non-IBM float formats needs to change.